lib/l10n/app_tr.arb

```
@@ -337,5 +338,7 @@
   },
   "appLogsFilteredCount": "{shown} / {total}",
   "invalidRelayUrlScheme": "URL wss:// veya ws:// ile başlamalıdır",
-  "invalidRelayUrl": "Geçersiz röle URL'si"
+  "invalidRelayUrl": "Geçersiz röle URL'si",
+  "helpAndFeedback": "Help and feedback",
```

RuboCop (1.85.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)